gems/smithy-client/lib/smithy-client.rb

```diff
@@ -1,18 +1,20 @@
 # frozen_string_literal: true
 
+require 'jmespath'
+
 # client
 
 require_relative 'smithy-client/block_io'
 require_relative 'smithy-client/configuration'
 require_relative 'smithy-client/handler'
+require_relative 'smithy-client/handler_context'
```

Comment:
Alphabetize under handler builder

Reply:
ah whoops, I think this came from bad merges.  So many merge conflicts over time in this pr, haha.